src/index.js

```diff
@@ -12,22 +14,30 @@ class Saturn {
    * @param {string} [opts.cdnURL=saturn.ms]
    * @param {number} [opts.connectTimeout=5000]
    * @param {number} [opts.downloadTimeout=0]
+   * @param {string} [opts.orchUrl]
+   * @param {import('./utils/storage.js').Storage} [opts.storage]
    */
   constructor (opts = {}) {
     this.opts = Object.assign({}, {
       clientId: randomUUID(),
       cdnURL: 'saturn.ms',
       logURL: 'https://twb3qukm2i654i3tnvx36char40aymqq.lambda-url.us-west-2.on.aws/',
       connectTimeout: 5_000,
-      downloadTimeout: 0
+      downloadTimeout: 0,
+      orchUrl: 'https://orchestrator.strn.pl/nodes?maxNodes=100'
```

Comment:
Lets move this next to the other 2 URL opts, and do `orchURL` instead of `orchUrl` for consistency.

Reply:
Done :)